package.json

```diff
@@ -126,7 +126,7 @@
     },
     "dependencies": {
         "@0x/assert": "^3.0.4",
-        "@0x/asset-swapper": "0xProject/gitpkg-registry#0x-asset-swapper-v4.6.0-a2f0d5eed",
+        "@0x/asset-swapper": "0xProject/gitpkg-registry#0x-asset-swapper-v4.6.0-db0f3a878",
```

Comment:
Should this contain the hash of the latest `master` when https://github.com/0xProject/0x-monorepo/pull/2684#pullrequestreview-478852431 is merged?

Reply:
Yup, done in https://github.com/0xProject/0x-api/pull/320/commits/c38fca77f088a4dbf328d17c194e02dffbb265ea